test/tools/libtesteth/TestOutputHelper.cpp

```diff
@@ -32,10 +32,10 @@ using namespace boost;
 
 Timer TestOutputHelper::m_timer;
 size_t TestOutputHelper::m_currTest = 0;
-size_t TestOutputHelper::m_maxTests = 0;
+size_t TestOutputHelper::m_maxTests = 1;
 string TestOutputHelper::m_currentTestName = "n/a";
 string TestOutputHelper::m_currentTestCaseName = "n/a";
-string TestOutputHelper::m_currentTestFileName = "n/a";
+string TestOutputHelper::m_currentTestFileName;
 std::vector<TestOutputHelper::execTimeName> TestOutputHelper::m_execTimeResults;
 void TestOutputHelper::initTest(int _maxTests)
```

Comment:
Use `size_t` type for argument if you want to pass results of `size()` functions.